apps/juxtaposition-ui/src/services/juxt-web/views/web/post.tsx

```diff
@@ -73,7 +73,7 @@ export function WebPostView(props: PostViewProps): ReactNode {
 				style={{
 					display: post.is_spoiler ? 'none' : undefined // Will be removed by spoiler-overlay onclick
 				}}
-				evt-click={`location.href='/posts/${post.id}`}
+				evt-click={`location.href='/posts/${post.id}'`}
```

Comment:
honestly why isn't this just an <a> tag. how did we get here
don't change this, it's how the original is, more venting that we should do that redesign and redo the HTML markup at the same time